src/docs/api/public.mdx

```diff
@@ -113,8 +132,21 @@ Specify the parameters using a list, and a serializer if needed.
 Specify the request and response serializers.
 
 - Request serializers, since they use DRF serializers, can generally just be the serializer itself. There is plenty of customization and if needed can be overriden with `inline_serializer`. These will be passed to the `request` parameter in `extend_schema`.
-- For our response serializers, you must type the `serialize` function's return with a `TypedDict` and pass the serializer as a parameter in `extend_schema`.
+- For our response serializers, you must type the `serialize` function's return with a `TypedDict` and pass the serializer as a parameter in `extend_schema`. See [here](https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/organization_member.py) for an example of this.
```

Comment:
There probably isn't much risk that the example will move out from under this link, but in case you want to hedge against it, we could link to the particular hash that's in the master branch right now.

Reply:
I'd like to keep it live for now, in case we add any more improvements to it, but will try and watch out for breaking this link.